README.md

```diff
@@ -1,2 +1,35 @@
 # ansible-cartodb
 Ansible role to build a multi-machine vagrant setup for CartoDB
+
+Currently targets tag 3.12.2 on January 29, 2016
+
+## Requirements
+
+Before running `vagrant up`, install the following on your host machine:
+  - vagrant 1.6+
+  - ansible 1.8.4+
+
+Then, clone the CartoDB repo to `../cartodb`:
+```
+git clone --recursive https://github.com/cartodb/cartodb ../cartodb
+```
+If this path is already in use for some reason, you can clone the repo elsewhere,
+and update the path in the Vagrantfile.
+
+Lastly, you'll need to add the following entries in your hosts file to point to the VM:
+```
+192.168.20.100 development.localhost.lan
```

Comment:
This might work, but would at the least require more configuration of cartodb. I'll tinker, but will likely not do this (although its nice to know about)